src/tests/Lotto.js

```diff
@@ -0,0 +1,34 @@
+import { ConstantNumbers } from './constant';
+
+const createLottoNumbersArray = () => {
+	return Array.from(
+		{ length: ConstantNumbers.MAX_NUMBER - ConstantNumbers.MIN_NUMBER + 1 },
+		(_, i) => i + ConstantNumbers.MIN_NUMBER
+	);
+};
+class Lotto {
+	static price = ConstantNumbers.LOTTO_PRICE;
+	lottoNumbers = createLottoNumbersArray();
```

Comment:
```suggestion
	static LOTTO_NUMBERS = createLottoNumbersArray();
```
마찬가지로 이렇게 상수로 만들어주면 어떨까 싶어요!